travel-event-host-app/src/models/user.ts

```diff
@@ -0,0 +1,21 @@
+import mongoose from 'mongoose';
+
+interface User {
+  firstName?: string;
+  imageUrl?: string;
+  lastName?: string;
+  email: string;
+  password: string;
+  location: {
+    country: string;
+    state: string;
+    city: string;
+    coords: {
+      lat: number;
+      long: number;
+    };
+  };
+  eventIds: mongoose.Types.ObjectId[];
+  admin?: boolean;
+}
```

Comment:
We could create another `type` in this file called `SecureUser` that will be the `User` just without the sensitive info. 
```
export type SecureUser = Omit<User, "email" | "password" | "admin">
```
Then in the GET user/:id route, you can create a typed `SecureUser` object and send that back instead of the entire User document

Reply:
great idea

Reply:
but we need to know if the user is a admin and the user email

Reply:
I will see if I can reference the user object without the sensitive data(for now only password)